### PR TITLE
feat: add alert banner for errors in sigin page

### PR DESCRIPTION
**Pull request by G3root**
[Screencast 2023-09-01 16:26:52.webm](https://github.com/documenso/documenso/assets/84864519/d05db848-e572-4dc9-b0f6-c83b0c5d6339)

### CODE REVIEW

File: packages/ui/primitives/alert.tsx [hunk 1]
```diff
@@ -5,21 +5,20 @@ import { VariantProps, cva } from 'class-variance-authority';
 import { cn } from '../lib/utils';
 
 const alertVariants = cva(
-  'relative w-full rounded-lg border p-4 [&>svg]:absolute [&>svg]:text-foreground [&>svg]:left-4 [&>svg]:top-4 [&>svg+div]:translate-y-[-3px] [&:has(svg)]:pl-11',
```
Comment (G3root) [inline packages/ui/primitives/alert.tsx:8]: had some alignment issues with the icon, copy and pasted the component from `Shadcn/ui`

**Comment by dephraiim** [comment]
The idea was to show a toast on error. Something like what dub.sh does.

https://github.com/documenso/documenso/assets/55143799/3928c6d6-4eb5-48ed-9e68-16c953dc0e15

**Comment by G3root** [comment]
@dephraiim updated the PR with the requested changes

[Screencast 2023-09-02 11:44:40.webm](https://github.com/documenso/documenso/assets/84864519/079d90ab-7fff-426a-8747-7c5b4a52d449)

File: apps/web/src/components/forms/signin.tsx [hunk 2]
```diff
@@ -39,6 +51,31 @@ export const SignInForm = ({ className }: SignInFormProps) => {
     resolver: zodResolver(ZSignInFormSchema),
   });
 
+  const timer = useRef<NodeJS.Timeout | null>(null);
+
+  useEffect(() => {
+    const error = searchParams?.get('error');
+    if (error) {
+      timer.current = setTimeout(() => {
+        // FIXME: Toast not firing without the TimeOut
```
Comment (dephraiim) [inline apps/web/src/components/forms/signin.tsx:60]: @Mythie This is what I talked about. @G3root solved it with a timeout, take it out and the toast fails to run but if you place a console log inside the useEffect, it runs on the page.

Comment (Mythie) [reply apps/web/src/components/forms/signin.tsx]: Gotcha, that was an annoying one. I tidied it a little but it's good to go!

**Comment by Mythie** [comment]
Let's ship it!

lgtm-b99

**Pull request merged**